timelock-agent/src/main/java/com/palantir/timelock/paxos/AsyncTimeLockServicesCreator.java

```diff
@@ -82,16 +83,16 @@ public TimeLockServices createTimeLockServices(
     private static AsyncTimelockService createRawAsyncTimelockService(
             String client,
             Supplier<ManagedTimestampService> timestampServiceSupplier) {
-        ScheduledExecutorService reaperExecutor = Executors.newSingleThreadScheduledExecutor(
-                new ThreadFactoryBuilder()
+        ScheduledExecutorService reaperExecutor = new InstrumentedScheduledExecutorService(
+                Executors.newSingleThreadScheduledExecutor(new ThreadFactoryBuilder()
```

Comment:
I'd suggest using `PTExecutors` instead of `Executors`, but it seems we don't actually use the tracing provided by `PTExecutors` currently... so I think it's fine to use executors for now. We should fix our tracing propagation across executors though, but better do it in one PR across the whole codebase.